SilMock/Google/Service/Directory/UsersResource.php

```diff
@@ -219,11 +218,10 @@ public function insert($postBody)
      */
     public function update($userKey, $postBody)
     {
-
         $userEntry = $this->getDbUser($userKey);
         if ($userEntry === null) {
             throw new \Exception(
-                "Account doesn't exist: " . $userKey,
+                "Account doesn't exist: " . json_encode($userKey, true),
```

Comment:
This was more-or-less a debugging change. Do you want to keep this call to `json_encode()` or remove it?

Reply:
Yes, I wanted to keep it.